src/sort.ts

```diff
@@ -45,7 +45,11 @@ function prepareDirection(direction: any = 1): SortDirectionForCmd {
     case '-1':
       return -1;
     default:
-      throw new MongoDriverError(`Invalid sort direction: ${JSON.stringify(direction)}`);
+      throw new MongoInvalidArgumentError(
+        `Invalid sort direction: ${JSON.stringify(
+          direction
+        )}\nSort direction must be one of 'ascending', 'asc', '1', 'descending', 'desc', '-1'`
```

Comment:
Maybe we can omit the possible correct values here. We stringify the input at the top of the function to make the switch statement simple but actually `1` and `-1` shouldn't be in quotes.